pkg/apps/manifest.json.in

```diff
@@ -18,8 +18,9 @@
     "content-security-policy": "img-src *",
 
     "config": {
-        "appstream_config_packages": @with_appstream_config_packages@,
-        "appstream_data_packages": @with_appstream_data_packages@
+        "debian": { "appstream_config_packages": ["appstream"] },
```

Comment:
This uses inverted keys compared to the storage page.  This page uses `config.OS.PARAMETER`, while the storage page uses `config.PARAMETER.OS`.  Both work of course, but is there a good reason to use one here and the other over there?

Reply:
Using `config.PARAMETER.OS` has the advantage that you can just set `config.PARAMETER = "foo"` in a override file and have things work even if `os-release` isn't useable.

Reply:
Right, I see. I felt it's a bit awkward to have the appstream_* key at the top, as you need a different config key for different OSes. But I do get the consistency argument. I'll rework it accordingly, and also test it for the "override without distro" case.

Reply:
Done now.